pages/ksphere/konvoy/latest/release-notes/index.md

```diff
@@ -15,8 +16,134 @@ enterprise: false
 
+### Version v1.5.0-beta.0 - April 9 March 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+|**Minimum** | 1.15.4 |
+|**Maximum** | 1.16.x |
+|**Default** | 1.16.8 |
+
+#### Improvements
+
+- New provisioner `gcp` to create clusters in [Google Cloud Platform](https://cloud.google.com/).
+- New flag `--max-parallel-nodes` when running `konvoy up`, `konvoy deploy` or `konvoy deploy kubernetes` to specify the number of nodes to upgrade in parallel. The value can be either an integer representing the number of nodes, or a percentage. Passing a value of `1` will upgrade the nodes serially.
+- Add validation for `control_plane_endpoint` to require a port number in the range of 1-65535 inclusive.
+- Only update yum cache when required to install RPM packages.
+- New option `spec.kubernetes.networking.iptables.addDefaultRules` that enables Konvoy to automatically setup iptables rules, required for the installation when the OS has strict iptables configuration.
+- Add a warning about the potentially destructive nature of the `--without-draining` flag.
+- Diagnostics bundle now contains the output of `helm ls` and `helm status <release deployment>`.
+- New Konvoy image tag with `ubi8` base image is now available and can be used with `export KONVOY_VERSION=<version>-ubi8`.
+- Set `imageID` in cluster.yaml for all provisioners instead of dynamically determining the image to use at runtime. This allows Konvoy to update the default imageID in the future without requiring users to modify the values in cluster.yam.
+
+#### Bug fixes
+
+- Import GPG keys before installing any RPM packages, which otherwise caused errors on any secured OS.
+- Fix a bug where certain addons were being installed even if set to `enabled: false`.
+- Fix a bug where provisioning would fail looking up the default AWS AMI in certain regions, even if `imageID` is set.
+- Fix a bug where the addon controller namespace is reapplied every time Konvoy runs, removing any metadata that might have been set.
+- Fix a bug with the airgapped artifact where `konvoy init` would fail with `Error: unexpected client error: reference not found`, requiring users to run `cd kubernetes-base-addons && git checkout master`.
+- Fixed a bug that would fail to install on Ubuntu if containerd.io was not already installed.
+
+#### Addons Improvements
+
+- Addons that require a different namespace than `kubeaddons` will have those namespaces created.
+- Improved error output when addons cannot be retrieved.
+
+#### Component version changes
+
+- Kubernetes `v1.16.8`
+- Calico `v3.13.1`
+- Go `1.13.8`
+- Ansible `2.7.16`
+
+#### Addon versions
+
+-   awsebscsiprovisioner: 0.5.0-1
+    - awsebscsiprovisioner: 0.5.0
```

Comment:
so many of these are single item addons that have the base-tech version shown in the revision. Doesn't have to be this release but maybe for future we want to show only the revision (eg `0.5.0-1`) if the addon is only made of one chart/app?